src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -29,38 +29,35 @@ const defaults = {
   disabled: false,
 }
 
-export const RadioGroup = ({
+export const RadioGroup = <T, >({
```

Comment:
why a comma here?

```suggestion
export const RadioGroup = <T>({
```

Reply:
otherwise the editor complains. It confuse it with an html element

Reply:
![image](https://github.com/user-attachments/assets/63dee867-853b-4944-89a2-6f90b93a801f)